config/initializers/figaro.rb

```diff
@@ -1,14 +1,9 @@
 Figaro.require_keys(
-  'DASHBOARD_SECRET_KEY_BASE',
-  'IDP_SLO_URL',
-  'IDP_SP_URL',
-  'IDP_SSO_URL',
-  'LOGIN_DASHBOARD_TOKEN',
-  'NEW_RELIC_LICENSE_KEY',
-  'SMTP_ADDRESS',
-  'SMTP_DOMAIN',
-  'SMTP_PASSWORD',
-  'SMTP_USERNAME',
-  'SP_NAME',
-  'SP_PASS',
+  'saml_idp_fingerprint',
```

Comment:
Did we already commit a version of these that's lowercase? We kept the casing upper to match existing env vars in https://github.com/18F/identity-dashboard/pull/70#issuecomment-283431625 on purpose